bindings/python/triplestore/backends/rdflib.py

```diff
@@ -0,0 +1,108 @@
+import warnings
+from typing import TYPE_CHECKING
+
+import rdflib
+from rdflib import BNode, Graph, URIRef
+
+from dlite.triplestore import Literal
+
+if TYPE_CHECKING:  # pragma: no cover
+    from collections.abc import Sequence
+    from typing import Generator
+
+    from dlite.triplestore import Triple
+
+
+def asuri(v):
+    """Help function converting a spo-value to proper rdflib type."""
```

Comment:
What is an 'spo-value' what is a 'proper' (as opposed to 'improper') rdflib type?

Reply:
It is an abbreviation for subject-predicate-object (i.e. an RDF triple). "spo" is common jargon in the field.